<h2>Description</h2>
<p>
  Cactus reference-free alignments of GRCh38 and T2T CHM13 v2.0, using chimp
  (GCF_002880755.1/panTro6) as an out-group.
</p>

<h2>Display Conventions and Configuration</h2>
<p>
  This track uses the <a href="/goldenPath/help/hgTracksHelp.html#SnakeDisplay">Snake tracks</a> display conventions and configuration.
</p>

<h2>Methods</h2>
<p>
  Alignments were generated using the Cactus reference alignment package and are stored
  as a HAL file.
</p>

<h2>Credits</h2>
<p>
  This track was created by <a href="mhauknes@ucsc.edu"> Marina Haukness &lt;mhauknes@ucsc.edu&gt;</a>
  if the UC Santa Cruz Computational Genomics Lab.
</p>

<h2>References</h2>
<p>
Armstrong J, Hickey G, Diekhans M, Fiddes IT, Novak AM, Deran A, Fang Q, Xie D, Feng S, Stiller J
<em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/33177663" target="_blank">
Progressive Cactus is a multiple-genome aligner for the thousand-genome era</a>.
<em>Nature</em>. 2020 Nov;587(7833):246-251.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33177663" target="_blank">33177663</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7673649/" target="_blank">PMC7673649</a>
</p>
<p>
Hickey G, Paten B, Earl D, Zerbino D, Haussler D.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/23505295" target="_blank">
HAL: a hierarchical format for storing and analyzing multiple genome alignments</a>.
<em>Bioinformatics</em>. 2013 May 15;29(10):1341-2.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23505295" target="_blank">23505295</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3654707/" target="_blank">PMC3654707</a>
</p>



